### PR TITLE
Structured text for landing pages

### PR DESCRIPTION
**Pull request by lsarni**
Mainly worked on the Structured Text for these pages:
- [Work At](https://deploy-preview-626--voorhoede-website.netlify.app/en/work-at-new?preview=true&previewSecret=blue-unsalted-ravioli)
- [About Us](https://deploy-preview-626--voorhoede-website.netlify.app/en/about-us-new?preview=true&previewSecret=blue-unsalted-ravioli)

Created the following Structured Text blocks:
- Blue text: structured text where all the text is blue and centered (headings are always blue, even outside this block)
- CTA list: list of links that are displayed as buttons
- Highlighted list: list of structured text blocks with a white background

What do you think of the names of the blocks in Dato?

### CODE REVIEW

File: src/components/app-core/typography.css [hunk 1]
```diff
@@ -1,11 +1,3 @@
-.hero {
```
Comment (lsarni) [inline src/components/app-core/typography.css:1]: Just deleting some styles that aren't used, could be on a different PR if you want.

File: src/pages/[language]/[slug]/index.query.graphql [hunk 2]
```diff
@@ -21,6 +21,7 @@ query LandingPage($locale: SiteLocale, $slug: String) {
         image {
           ...image
         }
+        imagePosition
```
Comment (lsarni) [inline src/pages/[language]/[slug]/index.query.graphql:24]: I didn't like the word `inverse` like we use in the old dato block, but I can also use that if you prefer

File: src/pages/[language]/[slug]/index.vue [hunk 3]
```diff
@@ -51,6 +52,12 @@
           external: cta.__typename === 'ExternalLinkRecord',
         }))"
       />
+      <structured-text-block
+        v-if="section.__typename === 'SectionStructuredTextRecord'"
+        :key="index"
```
Comment (Siilwyn) [inline src/pages/[language]/[slug]/index.vue:57]: Since the key is defined on the `v-for` think this `key` is not needed.

File: src/pages/[language]/[slug]/index.query.graphql [hunk 4]
```diff
@@ -62,6 +63,35 @@ query LandingPage($locale: SiteLocale, $slug: String) {
...
+              id
+              items {
+                body {
+                  value
+                }
+              }
+            }
+          }
+        }
+        gridAlignment
```
Comment (Siilwyn) [inline src/pages/[language]/[slug]/index.query.graphql:93]: Barely worth mentioning but if you agree let's put `gridAlignment` above `body`, tiny bit more readable I think.

File: src/pages/[language]/[slug]/index.query.graphql (hunk 2 above)
Comment (Siilwyn) [reply src/pages/[language]/[slug]/index.query.graphql:24]: Fully agree, in most situations I prefer to use 'options' over a single boolean because it allows to add more variants in the future.
